interface/patient_file/transaction/transactions_full.php

```diff
@@ -15,8 +15,9 @@
 
 <html>
 <head>
-<?php html_header_show();?>
```

Comment:
I wouldn't remove the above, since it seems that some folks use this as a hook for custom stuff.